tests/inference_on_device_test.py

```diff
@@ -36,6 +36,8 @@
 from sbi.simulators import diagonal_linear_gaussian, linear_gaussian
 from sbi.utils.torchutils import BoxUniform, gpu_available, process_device
 from sbi.utils.user_input_checks import (
+    process_prior,
+    process_simulator,
```

Comment:
why are those needed? Did you run `ruff` (it should remove unused imports)?

Reply:
I thought I did, my bad.